independent-publisher-connectors/RestCountries/apiDefinition.swagger.json

```diff
@@ -0,0 +1,795 @@
+{
+    "swagger": "2.0",
```

Comment:
@siddharth-vaghasia Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@sriyen-msft  - thanks for the review, can you please check now..I had fixed and commited again..